src/code/providers/google-drive-provider.js

```diff
@@ -101,19 +93,27 @@ class GoogleDriveProvider extends ProviderInterface {
     this.client = client
     this.authToken = null
     this.user = null
+    this.apiKey = this.options.apiKey
     this.clientId = this.options.clientId
+    if (!this.apiKey) {
+      throw new Error((tr("~GOOGLE_DRIVE.ERROR_MISSING_APIKEY")))
+    }
     if (!this.clientId) {
       throw new Error((tr("~GOOGLE_DRIVE.ERROR_MISSING_CLIENTID")))
     }
-    this.scopes = this.options.scopes || [
+    this.scopes = (this.options.scopes || [
       'https://www.googleapis.com/auth/drive',
       'https://www.googleapis.com/auth/drive.install',
       'https://www.googleapis.com/auth/drive.file',
       'https://www.googleapis.com/auth/userinfo.profile'
-    ]
+    ]).join(" ")
```

Comment:
Yes, between v2 and v3 they changed the scopes from an array to a space delimited string (with no documentation about making this change!)